src/storage/txn/scheduler.rs

```diff
@@ -940,17 +928,14 @@ impl Default for SchedContext {
     fn default() -> SchedContext {
         SchedContext {
             stats: HashMap::default(),
-            processing_read_duration: SCHED_PROCESSING_READ_HISTOGRAM_VEC.local(),
-            processing_write_duration: SCHED_PROCESSING_WRITE_HISTOGRAM_VEC.local(),
-            command_keyread_duration: KV_COMMAND_KEYREAD_HISTOGRAM_VEC.local(),
             command_gc_skipped_counter: KV_COMMAND_GC_SKIPPED_COUNTER.local(),
             command_gc_empty_range_counter: KV_COMMAND_GC_EMPTY_RANGE_COUNTER.local(),
         }
     }
 }
 
 impl SchedContext {
-    fn add_statistics(&mut self, cmd_tag: &'static str, stat: &Statistics) {
+    fn add_statistics(&mut self, cmd_tag: String, stat: &Statistics) {
```

Comment:
why String here? do we need to allocate a String now?

Reply:
It is just a stack allocation. I will make a PR to keep current `&'static`. It needs to add new methods to the static-metric macro.

Reply:
https://github.com/pingcap/rust-prometheus/pull/183